certified-connectors/Certopus/apiDefinition.swagger.json

```diff
@@ -0,0 +1,321 @@
+                    "itemValuePath": "id",
+                    "itemTitlePath": "name",
+                    "itemsPath": "data"
+                  },
+                  "title": "Select Organisation"
+                },
+                "eventId": {
+                  "type": "string",
+                  "description": "eventId",
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetEvent",
+                    "itemValuePath": "id",
+                    "itemTitlePath": "title",
+                    "itemsPath": "data",
+                    "parameters": {
+                      "organisationId": {
+                        "parameterReference": "body/organisationId"
+                      }
+                    }
+                  },
+                  "title": "Select Event"
+                },
+                "categoryId": {
+                  "type": "string",
+                  "description": "categoryId",
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetCategory",
+                    "itemValuePath": "id",
+                    "itemTitlePath": "title",
+                    "itemsPath": "data",
+                    "parameters": {
+                      "organisationId": {
+                        "parameterReference": "body/organisationId"
+                      },
+                      "eventId": {
+                        "parameterReference": "body/eventId"
+                      }
+                    }
+                  },
+                  "title": "Select Category",
+                  "x-ms-visibility": "important"
+                },
+                "generate": {
+                  "type": "boolean",
+                  "description": "generate"
+                },
+                "publish": {
+                  "type": "boolean",
+                  "description": "publish"
+                },
+                "recipients": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "data": {
+                        "type": "object",
+                        "properties": {
+                          "{Name}": {
+                            "type": "string",
+                            "description": "{Name}"
+                          }
+                        },
+                        "description": "data"
+                      },
+                      "email": {
+                        "type": "string",
+                        "description": "email"
+                      }
+                    }
+                  }
+                }
+              },
+              "required": ["organisationId", "eventId", "categoryId"],
+              "x-ms-visibility": "important"
+            },
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
@VrajGohil it is not allowed to have a default response only. please define a proper response. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
I've added the response